verify/verify_test.go

```diff
@@ -43,8 +44,10 @@ import (
 )
 
 var (
-	signMu sync.Once
-	signer *test.AmdSigner
+	signMu       sync.Once
+	signer       *test.AmdSigner
+	disableCerts = flag.Bool("disable_backfill", true,
```

Comment:
nit: disableCerts sounds a bit odd and can be confusing. We do not "disable certs" here, but "disable fetching from KDS". Maybe rename to "useCache" or something similar?

Reply:
Done.